src/amo/api/index.js

```diff
@@ -13,10 +50,16 @@ export function submitReview({
   body,
   reviewId,
   ...apiCallParams
-}) {
+}: SubmitReviewParams): Promise<ApiReviewType> {
   return new Promise(
     (resolve) => {
-      const data = { rating, version: versionId, body, title };
+      const data = {
```

Comment:
Sure, I changed it. However, I feel like a short and concise variable name makes code easier to understand when it only spans 10 lines or so. Otherwise the verbosity can slow a reader down.